indra_db/databases.py

```diff
@@ -143,7 +143,8 @@ class formats(_map_class):
                 'October2018-linux', 'February2020-linux'],
```

Comment:
There's also a `April2020-linux` version for Sparser which I believe is already in the indra_docker, should I add it in this list?

Reply:
Sure.